dashboard/app/models/objective.rb

```diff
@@ -28,4 +28,15 @@ class Objective < ActiveRecord::Base
   def summarize_for_edit
     {id: id, description: description}
   end
+
+  def summarize_for_lesson_show
+    {id: id, description: localized_description}
+  end
+
+  private
+
+  def localized_description
+    # TODO: localize the descriptions
```

Comment:
do you have a task in jira to keep track of this for later?

Reply:
I would think it's part of https://codedotorg.atlassian.net/browse/FND-1228, though that might just be a blocker for this.

Do we have a general catch-all task for localization of content?

Reply:
Ah it's here: https://codedotorg.atlassian.net/browse/PLAT-230. I'll update this method name to reflect the practices in that ticket

Reply:
Changes looks good! I updated the contents of that jira item slightly:
![Screen Shot 2020-10-28 at 11 43 28 AM](https://user-images.githubusercontent.com/8001765/97482104-efe80c80-1912-11eb-9242-c11d476a01fe.png)

Previously there was a bit of a conflict in the previous bullets over whether `localized_` was an acceptable prefix. Please let me know if the new wording + policy seems unclear or not right. in this case, it seems fine to err on the side of `display_` because it is a markdown field, even if we do not anticipate having resource keys, etc. in there.

Reply:
Yeah, I'm not sure we'll have have any markdown that needs transformation in that field, but it seems like either is okay in this case? I have no personal preference between `localized_` and `display_` here.

Reply:
ok, great. either way, this PR looks good!